docs/data/joy/components/typography/typography.md

```diff
@@ -13,6 +13,8 @@ githubLabel: 'component: Typography'
 
 The Typography component helps maintain a consistent design by providing a limited set of values to choose from and convenient props for building common designs faster.
 
+{{"demo": "TypographyUsage.js"}}
```

Comment:
```suggestion
{{"demo": "TypographyUsage.js" "hideToolbar": true, "bg": "gradient"}}
```

Just so we follow the introduction demo pattern 🤙

Reply:
Awesome! I saw this and wasn't sure how it should be configured.